### ISSUE TITLE
Turn off mouse debug info

### ISSUE THREAD

**Issue by C4Phone** [TakefiveInteractive/TedkOS#70]
It seems that on the demo day we added too much debug info. Mouse should not be tracked under text mode. It caused the system to crash.
![screen shot 2015-12-13 at 2 43 37 pm](https://cloud.githubusercontent.com/assets/7296488/11769245/ec426554-a1a7-11e5-9f92-a8ee8806dc25.png)

**Comment by C4Phone** [comment]
Caused by using both Maybe and nullptr to determine whether getElementAtPosition succeeded.

@tengyifei which one should we use?

![screen shot 2015-12-13 at 2 57 09 pm](https://cloud.githubusercontent.com/assets/7296488/11769305/d40f9004-a1a9-11e5-94ef-e2363be1d107.png)

![screen shot 2015-12-13 at 2 56 38 pm](https://cloud.githubusercontent.com/assets/7296488/11769307/d90e5f5e-a1a9-11e5-9a31-09db3f64448b.png)

**Comment by tengyifei** [comment]
There must be at least one element, the desktop. Therefore `getElementAtPosition` must always succeed, unless you are calling it with out-of-bounds coordinates or before `drawNikita`

**Comment by C4Phone** [comment]
Anyhow the system should crash due to Maybe unboxing, not due to dereferencing nullptr...

**Comment by C4Phone** [comment]
Github automatically closed this...

**Comment by C4Phone** [comment]
Mouse can also go out of bound in Video mode...

**Comment by Tedko** [comment]
Will it lead to crash?

**Comment by C4Phone** [comment]
No. I added some conditions. But it will print error message into text
terminal
